src/app/containers/Brand/index.jsx

```diff
@@ -3,9 +3,23 @@ import Brand from '@bbc/psammead-brand';
 import { ServiceContext } from '../../contexts/ServiceContext';
 
 const BrandContainer = () => {
-  const { brandName, brandSVG } = useContext(ServiceContext);
+  const { brandName, brandSVG, link } = useContext(ServiceContext);
+  const svgMaxHeight = 24;
+  const svgMinHeight = 16;
+  const svgRatio = brandSVG && brandSVG.ratio;
```

Comment:
Added this check because test withPageWrapper.test.jsx was failing

Reply:
I don't think we need that check either. You could pass `service` to  `<PageWrapperHOC />`. `withPageWrapper` is failing because no service was passed to `PageWrapperHOC` it therefore defaults to the default service config. The `default` config doesn't have `BrandSVG` defined. `configs` are in `lib/services`

Reply:
Hmmh, the news `service` is currently being passed to `<PageWrapperHOC />` through dataProps,  but (without the check to see if brandSVG is defined) - the test still fails

Reply:
You need to pass it on the component `<PageWrapperHOC data={dataProps} service="news" />`

Reply:
`PageWrapperHOC` returns `defaultPageWrapper` which expects service as a prop.

Reply:
👍 Made the change